boards/luatos/esp32s3_luatos_core/board.yml

```diff
@@ -0,0 +1,9 @@
+boards:
+  - name: esp32s3_luatos_core
```

Comment:
same comment regarding two boards vs. using a variant.

One reason for variants in HWMv2 are use-cases like this.